packages/react-avatar/src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { Badge } from '../Badge/Badge';
+import { StoryExample } from '../utils/StoryExample';
+import { SkypeCheckIcon, SkypeArrowIcon, SkypeMinusIcon, SkypeClockIcon } from '@fluentui/react-icons';
+
+export const BadgeExamples = () => (
+  <>
+    <StoryExample title="Badge">
+      <Badge size="smallest" state="error" />
```

Comment:
Just some thoughts:

I hadn't bothered questioning this earlier, but v0 refers to the Status mode as `state` prop. First, we should probably question that. The term is a bit overloaded with component state, and we really should consider a name more relevant to the purpose. Maybe `status` or `variant` even, where additional variants could be provided through theme.

So, what is a generic `Badge` component and when should it be used? Right now it represents generic status enum values which map to colors. However the spec assumes these are more about online presence. Would we have something like `PresenceBadge` with status enum values like `online` or `outOfOffice` or `offline`? Because I'm seeing things like the "no" icon used for offline which doesn't map to any implication in the Badge component.

For other types of Avatars, will they have different Badges? We've seen LinkedIn badges be represented as a LinkedIn image. Maybe for email clients, the badge might represent if the user is in your contacts list or not.

Reply:
(I totally do agree that `Avatar` should have a `badge` slot. Makes sense. Just wondering about what goes in that and what the default Avatar should do here.

Reply:
I like the idea of having a very basic/generic `Badge` component (or `useBadge` component hook) that can render any given color, icon, and shape (border radius), and uses the size given by the `--badge-size` token (which the Avatar's styles can set the value of).

Then we can have specialized badges like `PresenceBadge` or `StatusBadge` that has a domain-specific enum that maps to specific colors and icons: `badge={{ as: PresenceBadge, status: 'offline' }}`

Or, plot twist: maybe `Badge` is just a smaller `Avatar`? Perhaps that's going too far, but they would basically be the same thing: they can render an image or a colored background with an optional icon...

Reply:
That said, I'd rather address this in a follow-up PR and get this one checked in sooner than later.